pages/administratorCourseRequests/courseRequestsTable.ejs

```diff
@@ -33,7 +35,9 @@
                     <tr>
                         <td class="align-middle"><%= req.short_name %></td>
                         <td class="align-middle"><%= req.title %></td>
-                        <td class="align-middle"><%= req.user_name %></td>
+                        <td class="align-middle"><%= req.first_name + ' ' + req.last_name %></td>
```

Comment:
I'd consider including the `user.name` value here, just to see if it matches what they're self-reporting?
```suggestion
                        <td class="align-middle"><%= req.first_name + ' ' + req.last_name %> (<%= req.user_name %>)</td>
```